solr/core/src/test/org/apache/solr/search/TestDocSet.java

```diff
@@ -533,21 +535,21 @@ private void doTestIteratorEqual(Bits bits, Supplier<DocIdSetIterator>... disiSu
     }
   }
 
+
   /**
    * Tests equivalence among {@link DocIdSetIterator} instances retrieved from {@link BitDocSet} and {@link SortedIntDocSet}
-   * implementations, via {@link DocSet#getTopFilter()}/{@link Filter#getDocIdSet(LeafReaderContext, Bits)} and directly
-   * via {@link DocSet#iterator(LeafReaderContext)}.
-   * Also tests corresponding random-access {@link Bits} instances retrieved via {@link DocSet#getTopFilter()}/
-   * {@link Filter#getDocIdSet(LeafReaderContext, Bits)}/{@link DocIdSet#bits()}.
+   * implementations, via {@link DocSet#makeQuery()} and directly via {@link DocSet#iterator(LeafReaderContext)}.
+   * Also tests corresponding random-access {@link Bits} instances retrieved via {@link DocSet#makeQuery()}/
+   * {@link DocIdSet#bits()}.
    */
   public void doFilterTest(IndexReader reader) throws IOException {
     IndexReaderContext topLevelContext = reader.getContext();
     FixedBitSet bs = getRandomSet(reader.maxDoc(), rand.nextInt(reader.maxDoc()+1));
     DocSet a = new BitDocSet(bs);
     DocSet b = getIntDocSet(bs);
 
-    Filter fa = a.getTopFilter();
-    Filter fb = b.getTopFilter();
+//    Query fa = a.makeQuery();
+//    Query fb = b.makeQuery();
```

Comment:
This feels like we're now testing less than we were before. Can you explain why your patch removes testing without adding in equivalent functionality?